activesupport/lib/active_support/core_ext/object/inclusion.rb

```diff
@@ -12,6 +12,15 @@ class Object
   #
   # For non +Range+ arguments, this will throw an +ArgumentError+ if the argument
   # doesn't respond to +#include?+.
+  #
+  # As Active Record relations respond to +include?+, calling +in?+ will
+  # perform a query if the relation hasn't been loaded yet:
+  #
+  #   mary = Person.create(name: "Mary")
+  #   => #<Person id: 654, name: "Mary">
+  #   mary.in?(Person.where(name: "Alice"))
+  #   # SELECT 1 AS one FROM "people" WHERE "people"."name" ="Alice" ...
+  #
```

Comment:
Sounds weird to me to mention Active Record in Active Support documentation.

I understand how one could be bit by this, but it should make sense to users without the need to point it out.

Reply:
Ah Matthew already pointed it out, I should read the issues before commenting :facepalm: